README.md

```diff
@@ -193,15 +197,40 @@ treasury template --src /tmp/template.tpl --dst /tmp/result --append key1:v2
 ```
 This command ends up with output file where the value of variable key1 has a string "v2" appended.
 
+#### Template usage with variables interpolation
+
+Example template:
+
+```
+APPNAME={{ .AppName }}
+API_PASSWORD={{ .ApiPassword }}
+```
+
+```bash
+treasury template --src /tmp/template.tpl --dst /tmp/result.env --env AppName=test,ApiPassword=qwerty12345
+```
+
 Supported actions:
 
-#### read 
-Returns single value for given key
+#### read
+**DEPRECATED (please use [readFromEnv](#readfromenv))** Returns single value for given key
 
 ```
 {{ read "ENVIRONMENT/APPLICATION/SECRET_NAME" }}
 ```
 
+#### readFromEnv
+
+Returns single value for given key in specified environment
+
+```
+{{ readFromEnv "ENVIRONMENT" "APPLICATION/SECRET_NAME" }}
+
+# example using interpolation:
+
+{{ readFromEnv .Environment "app/API_PASSWORD" }}
+```
+
 Example:
 ```
 COCKPIT_API_PASSWORD={{ read "production/cockpit/cockpit_api_password" }}
```

Comment:
This should be updated

Reply:
Fixed in forcepushed version, good catch :+1: